web/src/components/apps/DashboardVersionCard.jsx

```diff
@@ -907,16 +909,28 @@ class DashboardVersionCard extends React.Component {
       return null;
     }
 
+    let latestVersion = downstream?.latestVersion;
+    let newVersionsAfterUpgrade = null;
+    let versionsToSkip = downstream?.pendingVersions?.length - 1;
+    const requiredVersions = downstream?.pendingVersions?.length > 0 ? filter(downstream.pendingVersions, ["isRequired", true]) : [];
+    if (this.props.currentVersion?.deployedAt && requiredVersions.length > 0) { // If there is a version already deployed, and there is at least one required pending versions, set latestVersion to the earliest required version
+      const sortedVersions = orderBy(requiredVersions, ["createdOn", "desc"]);
+      latestVersion = sortedVersions[0];
```

Comment:
wouldn't this get the latest required version instead of earliest?

Reply:
I'm not sure exactly why but using `asc` and `desc` return the same sort order for me. I'm going to update the method though to use `sortBy` which defaults to ascending order and is also grabbing the correct required version.